src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+    production: false,
+    ROOT_URL: "http://localhost:81",
 };
```

Comment:
Shouldn't this information only be in the ''environment.prod.ts'' file and its example?